Ports/SDLPoP/patches/disable_options_loading.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/src/options.c b/src/options.c
+index 1bfe473..2743e8b 100644
+--- a/src/options.c
++++ b/src/options.c
+@@ -55,24 +55,24 @@ int ini_load(const char *filename,
+ 		return -1;
+ 	}
+ 
+-	while (!feof(f)) {
+-		if (fscanf(f, "[%127[^];\n]]\n", section) == 1) {
++	//while (!feof(f)) {
++		/*if (fscanf(f, "[%127[^];\n]]\n", section) == 1) {
```

Comment:
I'm guessing that the formatting issue that caused this patch is due to formatting strings of the form `fscanf(f, "%127s")`.

If I'm not mistaken, this is the "optional specifier for maximum field width" (as specified in C99 chapter 7.19.6.2 section 3). It would make sense that this doesn't work, since [`LibC/scanf.cpp`](https://github.com/SerenityOS/serenity/blob/master/Userland/Libraries/LibC/scanf.cpp#L387) doesn't seem to support this feature at all.

Is that right?
In which case, instead of commenting everything out, it would be cleaner to simply remove the numbers :
```suggestion
+		if (fscanf(f, "[%[^];\n]]\n", section) == 1) {
```

Reply:
Right, nevermind, this is a bit more complex than handling that, since formatting strings like `fscanf(f, "%[]qwe]")` or `fscanf(f, "%[^]qwe]")` don't seem supported by the code either (it should be matched as a scan list respectively containing and not containing ']', 'q', 'w' and 'e'). In this case, the only clean option seems to comment it out indeed.
I'll check if that's indeed the case once my serenity compiles again, and open a few issues about these if needed.

Reply:
Ive just tested the code with your suggestion and it compiles and runs fine if you only remove those numbers, will submit the updated patch in a bit. Thanks for the help!

Reply:
> Right, nevermind, this is a bit more complex than handling that, since formatting strings like `fscanf(f, "%[]qwe]")` or `fscanf(f, "%[^]qwe]")` don't seem supported by the code either (it should be matched as a scan list respectively containing and not containing ']', 'q', 'w' and 'e'). In this case, the only clean option seems to comment it out indeed.
> I'll check if that's indeed the case once my serenity compiles again, and open a few issues about these if needed.

tbh I have no idea what the `[^]qwe` format specifer would even do, never seen something like that haha. ill try and read up a bit on this type of format specifiers. Atm the code works pretty good so if it doesnt work its not trivial for the games functionality :^)